api/src/main/java/org/xwiki/apps/googleapps/GoogleAppsScriptService.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.apps.googleapps;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+
+@Component
+@Named("GAScriptService")
```

Comment:
Even if this is not stated as a best practice rule, there's no point in using `ScriptService` in the name of the script service. In velocity this will be called with `$services.GAScriptService`, notice the redundancy. I would simply name this `googleApps` and call it with `$services.googleApps`

Reply:
Thanks for the hint. This helps readability indeed.

Reply:
Thanks, good suggestion. Done.